.travis.yml

```diff
@@ -0,0 +1,10 @@
+language: scala
+scala:
+   - 2.12.4
+services:
+  - mysql
+before_install:
+  - mysql -e 'CREATE DATABASE moduledir;'
+env:
```

Comment:
I know travis has a way of encrypting private keys so that should be done rather than committing the keys to the repo.

Reply:
https://docs.travis-ci.com/user/encryption-keys/

Reply:
Once that's done you might want to revoke the keys here, just in case.

Reply:
Changed the travis file. Thanks!